src/mscorlib/shared/System/StringComparer.cs

```diff
@@ -170,6 +171,7 @@ public int GetHashCode(object obj)
     }
 
     [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     internal sealed class CultureAwareComparer : StringComparer
```

Comment:
> I think the general binary serialization support is already going limit the set of changes we can make so I will just have to get over that

Yes :frowning:  That's partially why we trimmed the support back to a core set of types that, from the data we have available, seems to represent the most important types used in this contexts.